.github/workflows/github-action-type.yaml

```diff
@@ -5,14 +5,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9"]
+        python-version: ["3.10"]
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: install gdal lib
```

Comment:
Again, we shouldn't need GDAL here. Do we even need `pip install -e` for lint and typing (perhaps for mypy)? Or could we just install without the default dependencies?

Reply:
I removed -e, but we still need GDAL it seems like.